baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -374,9 +374,35 @@ export class AssetServiceClient {
  *   The first element of the array is an object representing [BatchGetAssetsHistoryResponse]{@link google.cloud.asset.v1.BatchGetAssetsHistoryResponse}.
  *   Please see the
  *   [documentation](https://github.com/googleapis/gax-nodejs/blob/master/client-libraries.md#regular-methods)
+ *   for more details and examples.
+  * @returns {Promise} - The promise which resolves to an array.
+ *   The first element of the array is an object representing
+ *   a long running operation. Its `promise()` method returns a promise
+ *   you can `await` for.
+ *   Please see the
+ *   [documentation](https://github.com/googleapis/gax-nodejs/blob/master/client-libraries.md#long-running-operations)
```

Comment:
This duplicate the above `return` comments. Could you double check here?

Reply:
The following files has similar problem.

Reply:
I'm going to break this into another PR, I think you're right!